### DISCUSSION TITLE
【简化首页】能不能加个功能把第一排左边那个硕大的幻灯展示给嘎了

### DISCUSSION THREAD

**Discussion by qaqz111** [the1812/Bilibili-Evolved#4904] [功能建议 / Ideas]
感觉大部分时间都不会看里面的东西，又那么大一个好占地方。

现在是用 AdGuard 的过滤功能屏蔽的，但是不知道为啥我 adguard 的设置老是丢失保存不了，重启浏览器后自定义的过滤规则就被清空了。

**Comment by Geeyun-JY3** [comment]
#4852 已经在预览版实装了：https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.9.1-preview

**Reply by qaqz111** [reply]
了解了，谢谢。
